### DISCUSSION TITLE
Failed, When submit zip.

### DISCUSSION THREAD

**Discussion by fengxiuyaun** [bdd100k/bdd100k#304] [Q&A] [answered]
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/6481257/229699567-67a07257-ad33-4cc5-a33d-14d92ba9895d.png">

When I submit results(zip bdd100k_mot_results.zip *.json), Failed. HOW TO Do?

**Comment by fengxiuyaun** [comment]
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/6481257/229700628-d5dca9cc-3f8d-4250-a007-59cd00951151.png">

**Reply by fengxiuyaun** [reply]
@thomasehuang

**Reply by thomasehuang** [reply]
We are looking into this issue now, thanks for notifying us of the issue.

**Reply by siyuanliii** [reply, marked as answer]
We have solved this issue. Could you please rerun your submission?
